src/Client/Settings.java

```diff
@@ -404,6 +406,15 @@ public static void definePresets(Properties props) {
     ATTACK_ALWAYS_LEFT_CLICK.put("all", true);
     ATTACK_ALWAYS_LEFT_CLICK.put(
         "custom", getPropBoolean(props, "bypass_attack", ATTACK_ALWAYS_LEFT_CLICK.get("default")));
+    
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("vanilla", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("vanilla_resizable", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("lite", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("default", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("heavy", true);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("all", true);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put(
+        "custom", getPropBoolean(props, "keep_magic_prayer_tab_cursor", KEEP_MAGIC_PRAYER_TAB_CURSOR.get("default")));
```

Comment:
this prop boolean would also need to be refactored ideally

Reply:
fixed